From 653b05bf9f5bf316efbacdbcb13049ca67acc032 Mon Sep 17 00:00:00 2001
From: Dongjin Kim <tobetter@gmail.com>
Date: Thu, 13 May 2021 11:56:17 +0900
Subject: [PATCH 105/109] Revert "ODROID-N2: arm64/dts: Move gpio-line-names
 into the periphs_pinctrl node"

This reverts commit 0142b646784e944d23ccabef00f129050d97cf2b.
---
 arch/arm64/boot/dts/amlogic/meson-g12b-odroid-n2.dts | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12b-odroid-n2.dts b/arch/arm64/boot/dts/amlogic/meson-g12b-odroid-n2.dts
index bcbd0b42516d..50c05f1674fe 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12b-odroid-n2.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-g12b-odroid-n2.dts
@@ -450,7 +450,7 @@ &frddr_c {
 	status = "okay";
 };
 
-&periphs_pinctrl {
+&gpio {
 	gpio-line-names =
 		/* GPIOZ */
 		"", "", "", "", "", "", "", "",
@@ -496,9 +496,6 @@ &periphs_pinctrl {
 		"PIN_3",  /* GPIOX_17 */
 		"PIN_5",  /* GPIOX_18 */
 		"PIN_36"; /* GPIOX_19 */
-};
-
-&gpio {
 	/*
 	 * WARNING: The USB Hub on the Odroid-N2 needs a reset signal
 	 * to be turned high in order to be detected by the USB Controller
-- 
2.25.1

